scripts/s3/s3api/copy_object.py

```diff
@@ -66,11 +64,12 @@ def __init__(self, access_key: str, secret_key: str, endpoint_url: str, test_id:
     @classmethod
```

Comment:
Why is this class method if we are creating instance of this class for each test_id? It should have been an instance method.

Reply:
It's raising pylint issue 'attribute defined outside init (W0201)'

Reply:
Then why not define those in `__init__` method?

Reply:
To avoid 'too many instance attributes (R0902)'

Reply:
@DhobaleVishal if we are using this only once, you can keep it as part of execute_copy_object_workload only.

Reply:
@priyankaborawake Yes, I did that, but it was introducing codacy issues W0201, so to avoid that added this method.